src/analysis/retail/monk/mistweaver/modules/spells/EssenceFont.tsx

```diff
@@ -335,6 +346,7 @@ class EssenceFont extends Analyzer {
             <ul>
               <li>{this.gomEFHits} additional hits</li>
               <li>{formatNumber(this.gomHealing)} additional healing</li>
+              <li>Essence Font Proc Ratio: {this.efProcRatio.toPrecision(2)}</li>
```

Comment:
```suggestion
              <li><SpellLink spell={TALENTS_MONK.ESSENCE_FONT}/> Proc Ratio: {this.efProcRatio.toPrecision(2)}</li>
```
also maybe explain what this is, I don't really know what proc ratio means without reading the code